src/db-service.ts

```diff
@@ -27,6 +27,25 @@ export class CategoryService {
     );
   }
 
+  /**
+   * Get all displine categories
+   */
+   async getAllDisplines(
+    dbHandler: TrxHandler,
+  ): Promise<Category[]> {
+    return (
+      dbHandler
+        .query<Category>(
+          sql`
+        SELECT *
+        FROM category_discipline
```

Comment:
Do you think it's possible to factor out the table name in a constant variable?

Reply:
I try to use a variable here ${table_name}, but it throw an error for that. I searched a bit online and ppl say that the table name should be static. If you know any way to set the table name as a variable, please let me know so that I don't have to do everything twice